### DISCUSSION TITLE
There is any way to update dictionary with daily newspeaper news: nouns, people, footbal player of the day and similar?

### DISCUSSION THREAD

**Discussion by vtontodonato** [openai/whisper#969] [Q&A] [answered]
there is any way to update the dictionary on a daily base using a king of "words of the day list" caming from major news sources (sport, events, etc.)

**Comment by glangford** [comment, marked as answer]
If `whisper` isn't transcribing correctly, you can use the approach described here.
* https://github.com/openai/whisper/discussions/288
* https://github.com/openai/whisper/discussions/963
